generators/voxel_generator_test.cpp

```diff
@@ -44,6 +44,7 @@ void VoxelGeneratorTest::generate_block(VoxelBlockRequest &input) {
 			generate_block_flat(**input.voxel_buffer, input.origin_in_voxels, input.lod);
 			break;
 
+		default:
```

Comment:
Prefer `default` at the end with a `CRASH_NOW` inside

Reply:
`CRASH_NOW_MSG("Mode is unrecognized.");`

This will suppress the message, but still exit when in non-debug mode.

Reply:
Yeah so far I've been doing that in all switch statements, but maybe in the future I'll make another macro